scripts/core/ui/components/ListPage/generic-list-page-item-view-edit.tsx

```diff
@@ -88,11 +89,10 @@ export class GenericListPageItemViewEdit<T extends object> extends React.Compone
 
     handleFieldChange(field: string, nextValue: valueof<IProps<T>['item']>) {
         // using updater function to avoid race conditions
-
         this.setState((prevState) => {
-            const nextItem = {...prevState.nextItem};
-
-            set(nextItem, field, nextValue);
+            const nextItem = produce(prevState.nextItem, (item) => {
```

Comment:
Can you rename `item` to `draft`? I think it's a good idea to follow this naming convention so it's clear to all developers where immer specific things start and end. It's also useful for reviews - I need to bring it up as an issue if you mutate stuff in react code, except `immer` and that `draft` naming is a quick way to tell.